vk/server.go

```diff
@@ -32,10 +33,11 @@ func New(opts ...OptionsModifier) *Server {
 	router.useQuietRoutes(options.QuietRoutes)
```

Comment:
should internalRouter also be quiet route aware

Reply:
the `router` here is the built in vektor one. This is the one that gets used as internalRouter. I'll change the name to make this less confusing